desktop/apps/artwork/lib/backbone_react_view/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component }  from 'react'
+import ReactDOM from 'react-dom'
+import invariant from 'invariant'
+import { createBackboneView } from './backbone_view.jsx'
+import { isArray, isEmpty } from 'underscore'
+
+export default function backboneReactView(config) {
+  const {
+    shouldMount = false,
+    views
+  } = config
+
+  const isValid = isArray(views) && !isEmpty(views)
+
+  invariant(isValid,
+    '(future/path/to/lib.jsx) ' +
+    'Error creating ReactBackboneView: an array of `views` must' +
+    'be provided.'
+  )
+
+  return (Container) => {
+    class BackboneViewWrapper extends Component {
+
+      state = {
+        Views: []
+      }
+
+      /** Don't update instantiable Backbone.View classes within the render cycle */
+      shouldComponentUpdate = () => false
+
+
+      componentWillMount() {
+        this.setState({
```

Comment:
I have a future render-cycle "update" mechanism in mind -- this is a sort of placeholder so that I don't forget